libs/zmath/src/zmath.zig

```diff
@@ -4490,18 +4490,32 @@ fn floatToIntAndBack(v: anytype) @TypeOf(v) {
 test "zmath.floatToIntAndBack" {
     {
         const v = floatToIntAndBack(f32x4(1.1, 2.9, 3.0, -4.5));
-        try expect(approxEqAbs(v, f32x4(1.0, 2.0, 3.0, -4.0), 0.0));
+        try expectVecEqual(v, f32x4(1.0, 2.0, 3.0, -4.0));
     }
     {
         const v = floatToIntAndBack(f32x8(1.1, 2.9, 3.0, -4.5, 2.5, -2.5, 1.1, -100.2));
-        try expect(approxEqAbs(v, f32x8(1.0, 2.0, 3.0, -4.0, 2.0, -2.0, 1.0, -100.0), 0.0));
+        try expectVecEqual(v, f32x8(1.0, 2.0, 3.0, -4.0, 2.0, -2.0, 1.0, -100.0));
     }
     {
         const v = floatToIntAndBack(f32x4(math.inf(f32), 2.9, math.nan(f32), math.snan(f32)));
         try expect(v[1] == 2.0);
     }
 }
 
+pub fn expectVecEqual(expected: anytype, actual: anytype) !void {
+    const T = @TypeOf(expected, actual);
+    inline for (0..veclen(T)) |i| {
+        try std.testing.expectEqual(expected[i], actual[i]);
+    }
+}
+
+pub fn expectVecApproxEqAbs(expected: anytype, actual: anytype, eps: f32) !void {
+    const T = @TypeOf(expected, actual);
+    inline for (0..veclen(T)) |i| {
+        try std.testing.expectApproxEqAbs(expected[i], actual[i], eps);
+    }
+}
+
 pub fn approxEqAbs(v0: anytype, v1: anytype, eps: f32) bool {
```

Comment:
note: I left this in case of dependencies & to not break people, happy to remove if desired.

Reply:
Yes let's just remove it please. We can mark the PR as a breaking change.

Reply:
Done. I had to keep this for a negation check, so I renamed it to match the testing methods.